apps/site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -190,48 +190,33 @@ You now have a block package that you can provide to apps to use, by **publishin
 
 ## Publish
 
-Once you've built a block, you can add it to the [Hub](/hub), so that:
-
-- your block will have an instant online demo playground
-- your block will be searchable via our block API
-- you can claim your namespace in blockprotocol.org
-
-To publish a block on the Hub, the initial process is as follows:
-
-1.  Have your block source code available in a GitHub repository
-1.  Claim your username by signing up at [https://blockprotocol.org](/)
-1.  Fork the [`blockprotocol` repository](https://github.com/blockprotocol/blockprotocol)
-1.  Add a JSON file to the [hub folder](https://github.com/blockprotocol/blockprotocol/tree/main/hub), in a subfolder matching the username you registered in step one, preceded by the `@` symbol (e.g. `@myusername`)
-1.  [Create a pull request](https://github.com/blockprotocol/blockprotocol/compare) from your fork
-
-The JSON metadata file should look like this:
-
-```json5
-{
-  // REQUIRED - path to the repository with your block's source code
-  repository: "https://github.com/hashintel/hash.git",
-  // REQUIRED - commit hash (pins block to a specific version)
-  commit: "adb915bf8fc4f84e33a2cd21f217225e50c3d7fa",
-
-  // IF NEEDED - see below
-  distDir: "dist",
-  folder: "blocks/block-embed",
-  workspace: "@hashintel/block-embed",
-}
-```
+Once you've built a block, you can add it to the [Hub](/hub), so that your block will have an instant online demo playground,
+and will be searchable via our block API.
+
+To publish a block on the Hub take the following steps.
+
+### In your browser
 
-### My block source is in a subfolder
+- Sign up for an account on this site if you haven’t already – click 'sign up' in the top right
+- [Create an API key](https://blockprotocol.org/settings/api-keys and copy it
 
-- If it’s in a yarn workspace, please provide `workspace`
-- If it’s not in a yarn workspace, please provide its `folder` path
+### In your terminal
 
-### My block requires building from its source
+- Run `npm run build` or `yarn build` to create a production build of your block (it will appear in the `dist/` folder)
```

Comment:
```suggestion
- Run `npm run build` or `yarn build` to create a production build of your block (it will appear in the `dist` folder)
```

(to align with _Build_ section)